common/src/main/java/com/graphicsfuzz/common/util/UpgradeShadingLanguageVersion.java

```diff
@@ -74,37 +86,84 @@
    */
   private List<Stmt> globalVariableInitializers = new ArrayList<>();
 
+  // Variable name renaming map
```

Comment:
I would change the comment to say that it maps a variable from its original name to a new name (and similar for functionRename).

Reply:
Done.